config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users
 end
```

Comment:
[OPTIONAL]
Since we are not using all routes, we can easily define only the ones we need:
`resources :users, only: [:new, :create, :edit, :update] `